styles/styles.css

```diff
@@ -1086,9 +1086,13 @@ main.blue-contract .section.section-with-title p {
   justify-content: center;
   letter-spacing: 1.25px;
   line-height: 100%;
-  padding: 17px 36px;
-  text-align: center;
+  max-width: 21.5em;
```

Comment:
why is needed?

Reply:
😢 
<img width="548" alt="image" src="https://github.com/Netcentric/vg-volvotrucks-us-rd/assets/3090999/008746c8-9599-4d62-b367-6cbdcaab4727">